### PR TITLE
Added blacklisting transaction on the `TxPool`

### PR DESCRIPTION
**Pull request by xgreenx**

### CODE REVIEW

File: bin/fuel-core/src/cli/run/tx_pool.rs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+};
+
+#[derive(Debug, Clone, clap::Args)]
+pub struct TxPoolArgs {
+    /// The max time to live of the transaction inside of the `TxPool`.
+    #[clap(long = "tx-pool-ttl", default_value = "5m", env)]
+    pub tx_pool_ttl: humantime::Duration,
+
+    /// The max number of transactions that the `TxPool` can simultaneously store.
+    #[clap(long = "tx-max-number", default_value = "4064", env)]
```
Comment (bvrooman) [inline bin/fuel-core/src/cli/run/tx_pool.rs:19]: Where did the "4064" come from?

**Review by bvrooman — APPROVED** [review]
It sounds like every node will need to have the same blacklist in their startup args. Otherwise, some nodes will accept the TX, while some will not.

File: bin/fuel-core/src/cli/run/tx_pool.rs (hunk 1 above)
Comment (Voxelot) [reply bin/fuel-core/src/cli/run/tx_pool.rs:19]: looks like it was just copied over from before

**Review by Voxelot — APPROVED** [review]
lgtm! pushed a version bump commit as well so we can release this.

**Pull request merged**